addons/event_sale/models/sale_order.py

```diff
@@ -150,7 +150,7 @@ def get_sale_order_line_multiline_description_sale(self, product):
                 lang=self.order_id.partner_id.lang,
             )
 
-            return ticket._get_ticket_multiline_description() + self._get_sale_order_line_multiline_description_variants()
+            return ticket._get_ticket_multiline_description() + self.with_context(lang=self.order_id.partner_id.lang)._get_sale_order_line_multiline_description_variants()
```

Comment:
Why is this diff needed?
Now that `get_sale_order_line_multiline_description_sale` is called with the right lang, you don't have to modify anything :thinking: (and you could even remove the context change above)

Reply:
> Plus get_sale_order_line_multiline_description_sale does not always have the right language

Well, isn't it the point of this PR? to make sure it's always called with the right language ?
That way we won't need those multiple with_context in the method overrides, as it will be executed with the right context from the start :thinking:

Reply:
Well, whatever the flow (sale/website_sale), we want the whole `get_sale_order_line_multiline_description_sale` call to be done with a consistent language context no?  Either the partner lang or the lang already in the context (for ecommerce orders).

So we should only have `with_context(lang=...)` calls before the call to `get_sale_order_line_multiline_description_sale`, not inside or inside its overrides/called methods.

Reply:
Haaaa yes, I didn't see that here it was `_get_sale_order_line_multiline_description_variants` inside `get_sale_order_line_multiline_description_sale` and I got confused between the two. So yes the with_context should be at the `get_sale_order_line_multiline_description_sale` level and the modif of this line can be removed :sweat_smile: